inspektor/lint.py

```diff
@@ -45,10 +46,15 @@ def __init__(self, args, logger=logging.getLogger('')):
             self.log.info('Pylint enabled : %s' % self.enabled_errors)
         else:
             self.log.info('Verbose mode, no disable/enable, full reports')
+        help_result = process.run('pylint --help')
```

Comment:
Well this looks like a global thing and not per-object one. How about moving this to the top and evaluate only once per import?